packages/app/src/cli/utilities/app/http-reverse-proxy.test.ts

```diff
@@ -69,7 +70,7 @@ describe('runConcurrentHTTPProcessesAndPathForwardTraffic', () => {
     vi.mocked(getAvailableTCPPort).mockResolvedValueOnce(4000)
 
     // When
-    const got = await runConcurrentHTTPProcessesAndPathForwardTraffic(undefined, [], [])
+    const got = await runConcurrentHTTPProcessesAndPathForwardTraffic('', undefined, [], [])
```

Comment:
this function might improve if the params are grouped into a typed object, is weird to see `('', undefined, [], [])`
(Not necessarily needed for this PR, but if you can do it that'd be awesome)